src/app/campaign/store/actions/campaign-action.service.ts

```diff
@@ -95,8 +95,10 @@ export class CampaignActionService {
   }
 
   setFlightGoal(flight: Flight) {
-    const { id, totalGoal, dailyMinimum } = flight;
-    this.store.dispatch(new campaignActions.CampaignFlightSetGoal({ flightId: id, totalGoal, dailyMinimum, valid: !!totalGoal }));
+    const { id, totalGoal, dailyMinimum, uncapped } = flight;
+    // TODO: goals of 0 seem to be fine, even for capped flights
+    const valid = totalGoal >= 0 && dailyMinimum >= 0;
```

Comment:
Was thinking this through in the context of PRX/augury.prx.org#233 ... we don't actually need to prevent them from saving a goal of 0 (as a placeholder flight or something).  Augury already allows a goal of 0.

(And uncapped flights always have goal/daily-min of 0).